### PR TITLE
Removed workTypes from roles as specified

### PR DESCRIPTION
**Pull request by ben-james-hmcts**
### Jira link

https://tools.hmcts.net/jira/browse/COT-896

### Change description

Removed a number of workTypes from judiciary roles

### Testing done

Updated all relevant unit tests and functional tests

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [X] Does this PR introduce a breaking change

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolStcicJudicialRoleMappingTest.java [hunk 1]
```diff
@@ -34,25 +34,13 @@ class DroolStcicJudicialRoleMappingTest extends DroolBase {
...
-        expectedRoleNameWorkTypesMap.put("judge", "hearing_work,decision_making_work,routine_work,"
-                + "applications,priority");
+        expectedRoleNameWorkTypesMap.put("judge", "decision_making_work");
         expectedRoleNameWorkTypesMap.put("case-allocator", null);
         expectedRoleNameWorkTypesMap.put("task-supervisor", null);
         expectedRoleNameWorkTypesMap.put("hmcts-judiciary", null);
         expectedRoleNameWorkTypesMap.put("specific-access-approver-judiciary", "access_requests");
         expectedRoleNameWorkTypesMap.put("leadership-judge", null);
-        expectedRoleNameWorkTypesMap.put("fee-paid-judge", "hearing_work,decision_making_work,"
-                + "routine_work,applications,priority");
```
Comment (abul-syed) [inline src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolStcicJudicialRoleMappingTest.java:45]: for roleAssignments that don't require worktypes, maybe we should add roleName to map with values set to null- e.g. `expectedRoleNameWorkTypesMap.put("senior-judge", null);` ?

**Review by abul-syed — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolStcicJudicialRoleMappingTest.java (hunk 1 above)
Comment (ben-james-hmcts) [reply src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolStcicJudicialRoleMappingTest.java:45]: I have added all role names with null work types back to the map in the same order as the role mappings data.

**Review by abul-syed — APPROVED** [review]
lgtm

**Review by mattnayler — APPROVED** [review]
LGTM

**Pull request merged**